Documentation/README_PulseShader.md

```diff
@@ -2,22 +2,24 @@
 
 ![MRTK_SpatialMesh_Pulse](https://user-images.githubusercontent.com/13754172/68261851-3489e200-fff6-11e9-9f6c-5574a7dd8db7.gif)
 
+![MRTK_HandMesh_Pulse2](https://user-images.githubusercontent.com/13754172/68262035-e4f7e600-fff6-11e9-9858-796afd1cabc5.gif)
```

Comment:
Should these changes be part of the PR?

Reply:
No they should not, reverted these changes.